__tests__/Rules-equalsField.spec.tsx

```diff
@@ -1,50 +1,53 @@
-import { mount } from 'enzyme';
 import React from 'react';
+import { getValidState } from '../__test_utils__/getValidState';
 import TestInput from '../__test_utils__/TestInput';
 
 import Formsy from '../src';
-import { WrapperInstanceMethods } from '../src/withFormsy';
 
 function ValidationForm(props: { validations: string; value?: any; other?: any }) {
   const { validations, value, other } = props;
 
   return (
     <Formsy>
-      <TestInput name="foo" validations={validations} value={value} />
+      <TestInput name="foo" validations={validations} value={value} testId="test-input" />
       <TestInput name="other" value={other} />
     </Formsy>
   );
 }
 
-export function expectIsValid(testForm: React.ComponentElement<any, any>) {
-  const form = mount(testForm);
-  const inputComponent = form.find('Formsy(TestInput)').first();
-  return expect(((inputComponent.instance() as unknown) as WrapperInstanceMethods<any>).isValid());
-}
-
 describe('equalsField', () => {
-  it('should pass when the value is equal', () => {
-    expectIsValid(<ValidationForm validations="equalsField:other" value="bar" other="bar" />).toEqual(true);
+  it('should pass when the value is equal', async () => {
+    expect(await getValidState(<ValidationForm validations="equalsField:other" value="bar"
+                                               other="bar" />)).toEqual(true);
```

Comment:
This is very odd formatting to me, but that's the beauty of prettier, I don't have to care :)

Reply:
I've pushed a reformat with prettier.